src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/issuefinaldecision/esa/EsaIssueFinalDecisionAboutToSubmitHandlerTest.java

```diff
@@ -468,4 +476,132 @@ public void givenAnIssueFinalDecisionEvent_shouldNotUpdateStateAndDwpStateWhenSt
         assertThat(response.getData().getState(), is(State.WITH_DWP));
         assertThat(response.getData().getDwpState(), is(DwpState.DIRECTION_RESPONDED));
     }
+
+    @Test
+    public void givenAnIssueFinalDecisionEventIfHearingsIsNull_ThenDoNotSendHearingCancellationRequest() {
+        handler = new IssueFinalDecisionAboutToSubmitHandler(footerService, decisionNoticeService, validator,
+                hearingMessageHelper, true);
+
+        DocumentLink docLink = DocumentLink.builder().documentUrl("bla.com").documentFilename(String.format("Decision Notice issued on %s.pdf", LocalDate.now().format(DateTimeFormatter.ofPattern("dd-MM-YYYY")))).build();
```

Comment:
Could probably block this out to separate lines for each param. Same across other files too.